src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -367,6 +367,7 @@ subject_prefixes:
   - EFO
   - ICD10CM
   - ICD10WHO
+  - icd11.foundation
```

Comment:
## `mondo.sssom.config.yml`: `subject_prefixes`
I'm doing another quick check and re-running the build, but I added this missing namespace to `subject_prefixes`, ran a lexmatch script, and got what I expected.

I've added this as a step to `docs/developer/add-new-source.md`.

Reply:
Better avoid comments like this on specific PRs and make specific issues to discuss during our meetings.. Too much to unpack to answer.

Reply:
I think for this one you are right. I spoke w/ Trish a little bit about it already. I'll make an issue.